### PR TITLE
Conform to MediaWiki coding standards

### PR DESCRIPTION
**Pull request by samwilson**
This change introduces phpcs as a Composer script and as part of
the Travis tests, and fixes the machine-fixable standards
transgressions.

Refs https://phabricator.wikimedia.org/T145482

### CODE REVIEW

**Comment by samwilson** [comment]
I've fixed the remaining CS probs.

File: composer.json [hunk 1]
```diff
@@ -22,5 +27,12 @@
 		"branch-alias": {
 			"dev-master": "0.7.x-dev"
 		}
+	},
+	"scripts": {
+		"csfix": "phpcbf --standard=phpcs.xml",
```
Comment (legoktm) [inline composer.json]: Please use the standard names (https://github.com/wikimedia/mediawiki-tools-codesniffer#how-to-install). Notably, --standard= should not be necessary.

File: composer.json [hunk 2]
```diff
@@ -22,5 +27,12 @@
...
+		"cs": [
+			"parallel-lint . --exclude vendor",
+			"phpcs -p --standard=phpcs.xml"
```
Comment (legoktm) [inline composer.json]: Include -s so it outputs the sniff code.

**Comment by legoktm** [comment]
What's the minimum PHP version that addwiki requires? Is it at least 5.5?

**Comment by addshore** [comment]
Yes, the guzzle version used requires 5.5+

**Comment by samwilson** [comment]
Thanks @legoktm good points, I've fixed it up.

**Pull request merged**